dist/loginApp/loginApp.js

```diff
@@ -90,8 +90,8 @@
                         if (search.state) {
                             scope.changeCurrentForm(search.state);
                         }
-                        $location.search('app', null);
-                        $location.search('state', null);
+                        // $location.search('app', null);
```

Comment:
leave this comment for now. This removed the url parameters after reaching the page thus creating another state for the back button.